frontend/src/photoView/photoView.js

```diff
@@ -28,35 +29,105 @@ export class PhotoView extends React.Component {
         }
     }
 
-    render() {
-        if (this.state.loading) {
-            return (<>
+     flipPhoto = () => {
+         if (this.state.display_side === 'front') {
+             this.setState({
+                 display_side: 'back',
+             });
+         } else {
+             this.setState({
+                 display_side: 'front',
+             });
+         }
+     };
+
+     render() {
+         if (this.state.loading) {
+             return (<>
                 Loading!
-            </>);
-        }
-        if (!this.state.photo_data) {
-            return (<>
+             </>);
+         }
+         if (!this.state.photo_data) {
+             return (<>
                 Photo with id {window.location.pathname.split('/')[2]} is not in database.
-            </>);
-        }
-        const {
-            title,
-            alt,
-            front_src,
-            back_src,
-        } = this.state.photo_data;
+             </>);
+         }
+         const {
+             title,
+             alt,
+             front_src,
+             back_src,
+         } = this.state.photo_data;
 
-        return (<>
-            <Navbar />
-            <div style={{ padding: '100px 50px' }}>
-                <h2>{title}</h2>
-                <img width={500} height={500} src={front_src} alt={alt}/>
-                <img width={500} height={500} src={back_src} alt={alt}/>
-                <br/>
-            </div>
-            <Footer />
-        </>);
-    }
+         let src = front_src;
+         if (this.state.display_side === 'back') {
+             src = back_src;
+         }
+
+         return (<>
+             <Navbar />
+             <div>
+                 <div className='imageView' style={{ padding: '100px 50px', float: 'left' }}>
+                     <h2>Photo Title: {title}</h2>
+                     <img width={500} height={500} src={src} alt={alt}/>
+                     <br/>
+                     <button onClick={() => this.flipPhoto()}> Flip photo </button>
+                 </div>
+                 <div className='imageInfo'
+                     style={{ padding: '100px 50px', float: 'left', marginTop: 50 }}>
+                     <div className='attributeTitle' style={{ fontWeight: 'bold' }}>
+                        Photographer:
+                         <div className='attribute'style={{ fontWeight: 'normal' }}>
+                            None
+                         </div>
+                     </div>
+                     <div className='attributeTitle' style={{ fontWeight: 'bold' }}>
```

Comment:
I think _probably_ this repeated code is a sign that we need to refactor a bit. But let's take a look together in our call -- this is already good enough to get us to where we want to get with the prototype today!